ford/sourceform.py

```diff
@@ -43,7 +43,7 @@
 import ford.reader
 import ford.utils
 
-VAR_TYPE_STRING = "^integer|real|double\s*precision|character|complex|logical|type|class|procedure|enumerator"
+VAR_TYPE_STRING = "^integer|real|double\s*precision|character|double\s*complex|logical|type|class|procedure|enumerator"
```

Comment:
```suggestion
VAR_TYPE_STRING = "^integer|real|double\s*precision|character|complex|double\s*complex|logical|type|class|procedure|enumerator"
```
This will miss single precision complex without this change, no? You're applying `*` to `\s`. You could also add a zero or 1 modifier to the `double` part of the RE. I like adding an explicit single complex more, however.